python/tskit/trees.py

```diff
@@ -2370,6 +2370,67 @@ def kc_distance(self, other, lambda_=0.0):
         """
         return self._ll_tree.get_kc_distance(other._ll_tree, lambda_)
 
+    def split_polytomies(
+        self,
+        *,
+        epsilon=None,
+        method=None,
+        record_provenance=True,
+        random_seed=None,
+    ):
+        """
+        Return a new :class:`.Tree` where extra nodes and edges have been inserted
+        so that any any node ``u`` with greater than 2 children --- a multifurcation
+        or "polytomy" --- is resolved into successive bifurcations. New nodes are
+        inserted at times fractionally less than than the time of node ``u``
+        (controlled by the ``epsilon`` parameter).
+
+        If the ``method`` is ``"random"`` (currently the only option, and the default
+        when no method is specified), then for a node with :math:`n` children, the
+        :math:`(2n - 3)!!` possible bifurcating topologies are produced with equal
+        probability.
+
+        .. fixme:: Looks like this function is wrong
+
+        The returned :class`.Tree` will have the same genomic span as this tree,
+        and node IDs will be converved (that is, node ``u`` in this tree will
```

Comment:
converted

Reply:
The idea of node IDs being converted isn't obvious to me, could use another sentence of documentation. So if I had leaves 0, 1, 2, and a polytomy 3 that is the parent of all three leaves, then in the returned tree I would have 0, 1, 2, 3, 4? Where 3 is the "converted" node ID and 4 is an extra node from breaking the polytomy?